### ISSUE TITLE
Clean up mypy notes

### ISSUE THREAD

**Issue by vries** [nmoroze/tclint#150]
With mypy I get:
```
src/tclint/cli/tclsp.py:89: note: By default the bodies of untyped functions are not checked, consider using --check-untyped-defs  [annotation-unchecked]
src/tclint/cli/tclsp.py:91: note: By default the bodies of untyped functions are not checked, consider using --check-untyped-defs  [annotation-unchecked]
src/tclint/cli/tclsp.py:93: note: By default the bodies of untyped functions are not checked, consider using --check-untyped-defs  [annotation-unchecked]
src/tclint/cli/tclsp.py:96: note: By default the bodies of untyped functions are not checked, consider using --check-untyped-defs  [annotation-unchecked]
src/tclint/cli/tclsp.py:101: note: By default the bodies of untyped functions are not checked, consider using --check-untyped-defs  [annotation-unchecked]
```

It would be good to get rid of these, making it easier to spot if there's a new one.

**Comment by nmoroze** [comment]
Thanks for opening the issue. I discovered running with `--check-untyped-defs` bypasses this, and I cleaned up some of the errors that result in 7090405b9e6ec6587e0f12dfc94fdbf029c621ac. 

Once I wrap those up I'll enable this flag as a default. It would be good to annotate the functions as well, but I think it's better to make sure the additional type checking runs. It would have caught a recent bug (#148) statically.
